src/iconsManifest/supportedFolders.ts

```diff
@@ -511,6 +511,11 @@ export const extensions: IFolderCollection = {
       extensions: ['.venv', '.virtualenv'],
       format: FileFormat.svg,
     },
+    {
+      icon: 'vercel',
```

Comment:
you still need to provide an opened version of the folder. Look for other folders in the extension to see how it's done.